src/font.js

```diff
@@ -279,7 +279,8 @@ Font.prototype.defaultRenderOptions = {
          * and shouldn't be turned off when rendering arabic text.
          */
         { script: 'arab', tags: ['init', 'medi', 'fina', 'rlig'] },
-        { script: 'latn', tags: ['liga', 'rlig'] }
+        { script: 'latn', tags: ['liga', 'rlig'] },
+        { script: 'thai', tags: ['liga', 'ccmp'] },
```

Comment:
all font scripts are required to have rlig enabled by default. Please add rlig (Required Ligatures) feature.

Reply:
Feature has been added